lib/puppet/provider/service/redhat.rb

```diff
@@ -39,6 +41,7 @@ def enabled?
   # Don't support them specifying runlevels; always use the runlevels
   # in the init scripts.
   def enable
+      chkconfig("--add", @resource[:name])
```

Comment:
@joshcooper It looks like the network service is one of the only services left on RHEL 7 that you can control with chkconfig (and exists on v5 and v6 also). So I can refactor the acceptance test to use the network service instead of httpd (and be careful not to disable the service during the test!) :) Then we can run it on v5-v7.